.github/workflows/ci.yml

```diff
@@ -51,8 +53,6 @@ jobs:
           restore-keys: |
             composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('**/composer.*') }}-
             composer-${{ runner.os }}-${{ matrix.php-version }}-
-            composer-${{ runner.os }}-
-            composer-
```

Comment:
These are dangerous

Reply:
Could you exlain why? I think I've just copied these.

Reply:
On different PHP versions there are/may be different packages or package versions.

Reply:
True, but it's only the composer cache, not the `vendor` directory that's cached, no?

Reply:
It is true, `vendor`˛is not cached. Caching `vendor` is toxic. 🧪

Reply:
Yes, but then it shouldn't be an issue, no? Because if package versions differ, there's just one or two unnecessary packages in the cache that aren't used then but downloaded on demand.

Reply:
Yes. But I don't have the balls to cache things cross-os.

Reply:
I'm fine with either.

Reply:
Okay. Leave it like this and suffer +3 seconds of download time once now.